appuselfbot.py

```diff
@@ -36,17 +36,18 @@ def parse_cmd_arguments(): # allows for arguments
     parser.add_argument("-s", "--silent", # Allows for Testing of mac related code
                         action="store_true",
                         help="Supresses all errors")
-    parser.add_argument("--run-admin", # Allows the user to run the bot as admin or sudo
-                        action="store_true",
-                        help="Allows the bot to be run as admin")
+    #parser.add_argument("-a", "--admin", # Allows the user to run the bot as admin or sudo
+    #                    action="store_true",
+    #                    help="Allows the bot to be run as admin")
     return parser
 
 args = parse_cmd_arguments().parse_args()
 _test_run = args.test_run
 _force_mac = args.force_mac
 _reset_cfg = args.reset_config
 _silent = args.silent
-_force_admin = args.run_admin
+#_force_admin = args.admin

```

Comment:
Please remove the code instead of commenting them out; the nature of a git workflow makes all code removals easier to spot and revert, if necessary.